src/styles/lyrics.scss

```diff
@@ -0,0 +1,31 @@
+.lyricPage {
+    padding-top: 4.2em !important;
+    display: flex;
+    justify-content: center;
+}
+
+.dynamicLyricsContainer {
+    display: flex;
+    flex-direction: column;
+    height: max-content;
```

Comment:
This doesn't work on webOS 1.2 (old WebKit). You can reproduce the looking by removing `height: max-content;`.
The only way I found is to set `height` for the lyrics element.